File: app/src/main/java/com/socket/socket/entity/Utente.java
package com.socket.socket.entity;

public class Utente{
    private final String email;
    private final String password;

    public Utente(String email, String password){
        this.email = email;
        this.password = password;
    }

    public String getEmail(){
        return email;
    }

    public String getPassword(){
        return password;
    }
}

File: app/src/main/java/com/socket/socket/entity/Utente.kt
package com.socket.socket.entity

class Utente {
    private var username: String? = null
    private var password: String? = null

    constructor() {}
    constructor(username: String?, password: String?) {
        this.username = username
        this.password = password
    }

    fun getUsername(): String? {
        return username
    }

    fun getPassword(): String? {
        return password
    }
}